src/Orchard.Cms.Web/Modules/Orchard.Roles/Services/RoleStore.cs

```diff
@@ -58,10 +58,10 @@ public void UpdateRoles(RolesDocument roles)
             _memoryCache.Set(Key, roles);
         }
 
-        public async Task<IEnumerable<string>> GetRoleNamesAsync()
+        public async Task<IEnumerable<RoleNamesEntry>> GetRoleNamesAsync()
```

Comment:
All the code I saw was our code. This is a sample of use of IRoleProvider https://github.com/OrchardCMS/Orchard2/blob/openidUpgrade/src/Orchard.Cms.Web/Modules/Orchard.OpenId/Controllers/AdminController.cs#L107
The point is I need to store NormalizedName  here https://github.com/OrchardCMS/Orchard2/blob/openidUpgrade/src/Orchard.Cms.Web/Modules/Orchard.OpenId/Models/OpenIdApplication.cs#L63

Because I need to know the Normalized name for retrieving claims https://github.com/OrchardCMS/Orchard2/blob/openidUpgrade/src/Orchard.Cms.Web/Modules/Orchard.OpenId/Controllers/AccessController.cs#L345